app/pulp/app/serializers/progress.py

```diff
@@ -0,0 +1,57 @@
+from rest_framework import serializers
+
+from pulp.app import models
+from pulp.app.serializers import ModelSerializer
+
+
+class ProgressReportSerializer(ModelSerializer):
+
+    _href = serializers.HyperlinkedIdentityField(
+    view_name='progress-detail',
+    lookup_field='id',
+    )
+
+    message = serializers.CharField(
+        help_text="The message shown to the user for the progress update.",
+        read_only=True
+    )
+
+    state = serializers.CharField(
+        help_text="The current state of the progress update. The possible values include:"
+                  " 'waiting', 'skipped', 'running', 'completed', 'failed' and 'canceled'."
+                  " Default is waiting.",
+        read_only=True
+    )
+
+    total = serializers.IntegerField(
```

Comment:
This field will always be updated on the server side. Given that, this should get a `read_only=True` also right?